bridgeifns.py

```diff
@@ -0,0 +1,194 @@
+# stdlib
+from os import name
+from typing import Tuple
+# lib
+from cloudcix.rcc import comms_ssh, CouldNotConnectException
+# local
+
+
+__all__ = [
+    'build',
+    'scrub',
+]
+
+SUCCESS_CODE = 0
+
+
+def build(
+    bridgename: str,
+    namespace: str,
+) -> Tuple[bool, str]:
+    """
+    description:
+        Creates a veth link on the main namespace and connects it to a bridge.
+        Then, it moves one end of the link to a VRF network namespace and sets the interface up.
+
+    parameters:
+        bridgename:
+            description: The name of the bridge on the main namespace.
+            type: string
+            required: true
+        namespace:
+            description: The VRF network namespace identifier, such as 'VRF123'.
+            type: string
+            required: true
+
+    return:
+        description: |
+            A tuple with a boolean flag indicating if the veth link creation was successful,
+            and the output or error message.
+        type: tuple
+    """
+    # Define functions
+    def InterfaceExists(bridgename):
+        try:
+            exit_code, _, _ = comms_ssh(
+                host_ip='localhost',
+                payload=f'ip link show {bridgename}',
+                username='robot',)
+
+            if exit_code == 0:
+                return True
+            else:
+                return False
+
+        except CouldNotConnectException:
+            return False
+
+    # Define message
+    messages = {
+        1000: f'1000: Successfully created veth link and connected it to bridge {bridgename}, moved one end to namespace {namespace}.',
+        1001: f'1001: Failed to create veth link on bridge {bridgename}.',
+        1002: f'1002: Failed to set interface in namespace {namespace}.',
+    }
+
+
+    if InterfaceExists(bridgename) or InterfaceExists(namespace):
+        message = InterfaceExists(bridgename) *  f'Bridgename: {bridgename} already exists.' + InterfaceExists(namespace) * f'Namespace: {namespace} already exists.'
```

Comment:
All messages should be defined in the messages dict and appended to as in the example in previous comment. 

Can you explain what this if statement is for because it is not clear to me the objective.